cmd/external-plugins/cherrypicker/server.go

```diff
@@ -186,10 +186,15 @@ func (s *Server) handleIssueComment(l *logrus.Entry, ic github.IssueCommentEvent
 	})
 
 	cherryPickMatches := cherryPickRe.FindAllStringSubmatch(ic.Comment.Body, -1)
-	if len(cherryPickMatches) == 0 || len(cherryPickMatches[0]) != 2 {
+	if len(cherryPickMatches) == 0 || len(cherryPickMatches[0]) < 2 {
 		return nil
 	}
-	targetBranch := strings.TrimSpace(cherryPickMatches[0][1])
+	branches := strings.Fields(cherryPickMatches[0][1])
+	targetBranch := branches[0]
```

Comment:
Does this array access need a `len` check?

Reply:
I guess the match can't be an empty string...

Reply:
Yeah, the regex requires there to be a space for a match